.github/workflows/test-e2e.yml

```diff
@@ -1,4 +1,4 @@
-name: TestSuite E2E - create module
```

Comment:
this e2e test is only for create module, the execution time is relatively short (you don't need to provision a k3d cluster), please don't mix the deployment test with this one.

you can change and extend that smoke test (rename it) for this e2e task.